catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java

```diff
@@ -48,46 +43,95 @@ public class MysqlTableOperations extends JdbcTableOperations {
 
   @Override
   public JdbcTable load(String databaseName, String tableName) throws NoSuchTableException {
```

Comment:
seems PostgreSQL could resuse most code, is it possible to put the code in JdbcTableOperations? then PG, Snowflake could reuse it.